system-tests/lib/cre/environment/environment.go

```diff
@@ -466,100 +202,83 @@ func SetupTestEnvironment(
 			testLogger,
 			singleFileLogger,
 			&cre.WorkflowRegistryInput{
-				ContractAddress: common.HexToAddress(wfRegAddr),
-				ChainSelector:   homeChainOutput.ChainSelector,
-				CldEnv:          allChainsCLDEnvironment,
+				ContractAddress: common.HexToAddress(crecontracts.MustGetAddressFromDataStore(deployKeystoneContractsOutput.Env.DataStore, startBlockchainsOutput.BlockChainOutputs[0].ChainSelector, keystone_changeset.WorkflowRegistry.String(), input.ContractVersions[keystone_changeset.WorkflowRegistry.String()], "")),
```

Comment:
This would really benefit from being moved to a single component that configures our WorkflowRegistry, meaning:

- deployment
- configuring it <- the latter looks like it might have some duplicate CLD environment setup from other ops that use CLD; can we somehow abstract this out?